client/components/calypso-i18n-provider/index.tsx

```diff
@@ -2,21 +2,23 @@
  * External dependencies
  */
 import * as React from 'react';
-import { setLocaleData as setWpI18nLocaleData } from '@wordpress/i18n';
-import { I18nProvider } from '@automattic/react-i18n';
+import { createI18n, setLocaleData as setWpI18nLocaleData } from '@wordpress/i18n';
+import { I18nProvider } from '@wordpress/react-i18n';
 import i18n from 'i18n-calypso';
 import { LocaleProvider, i18nDefaultLocaleSlug } from '@automattic/i18n-utils';
 
 const CalypsoI18nProvider: React.FunctionComponent = ( { children } ) => {
 	const [ localeData, setLocaleData ] = React.useState( i18n.getLocale() );
```

Comment:
Not sure if introduced by this PR, but on my local machine I see a TypeScript error about `getLocale()`:

```
Property 'getLocale' does not exist on type 'typeof i18nCalypso'
```